### DISCUSSION TITLE
Enable 'mapping table' for Variable values

### DISCUSSION THREAD

**Discussion by MatinF** [grafana/grafana#46760] [Dashboards, Panels & Visualizations]
We are actively using Grafana Variables to toggle between what IoT device we display value from in our dashboards.

The IoT devices are identified through their "serial number" as in below screenshot. This is difficult to change in the code underlying our backend parser for the dashboard. However, we would very much like to be able to add a more meaningful name for each serialnumber, for example translating `2F6913DB` to `Truck 11B` (or showing both in the Variables dropdown).

Is this currently possible via some plugin or method? Otherwise we think this would be a great (and probably simple) addition to the Variables configuration page.

![image](https://user-images.githubusercontent.com/26184427/159122371-0899610a-7d9c-4bcd-8252-aa0dd0506bda.png)

Our Variables setup looks as below:

![image](https://user-images.githubusercontent.com/26184427/159122422-f3e4e6f3-5d00-4022-82b0-cd0411325b5a.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

Edit: I think this is being discussed here https://github.com/grafana/grafana/issues/1032
